src/server/session.ts

```diff
@@ -298,7 +298,7 @@ export function formatDiagnosticToProtocol(diag: Diagnostic, includeFileName: bo
         : common;
 }
 
-export interface PendingErrorCheck {
+interface PendingErrorCheck {

```

Comment:
This may be used? https://github.com/search?q=%2F%5Cbts%5C.server%5C.PendingErrorCheck%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code